### PR TITLE
changed isUUIDValidHandler to uuid validator

### PR DESCRIPTION
**Pull request by piotr-major-rmtcfm-com**

### CODE REVIEW

File: app/validators/uuidValidator.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+
+async function isUUIDValid(id) {
+    if (!validate(id)) {
+        return Promise.reject('Id must be valid UUID');
+    }
+}
+
+const id = [param('id').custom(id => isUUIDValid(id))];
+
+module.exports = { id };
```
Comment (daniel-pupczyk-rmtcfm-com) [inline app/validators/uuidValidator.js]: just `module exports = [param('id')...`

File: app/validators/uuidValidator.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+const { param } = require('express-validator');
+const { validate } = require('uuid');
+
+async function isUUIDValid(id) {
+    if (!validate(id)) {
+        return Promise.reject('Id must be valid UUID');
```
Comment (daniel-pupczyk-rmtcfm-com) [inline app/validators/uuidValidator.js]: why promis? you can return boolean

Comment (rafal-ksiazek-rmtcfm-com) [reply app/validators/uuidValidator.js]: I think we need to return promise or throw error

**Review by rafal-ksiazek-rmtcfm-com — APPROVED** [review]

File: app/validators/uuidValidator.js (hunk 2 above)
Comment (piotr-major-rmtcfm-com) [reply app/validators/uuidValidator.js]: yes - it should be promise

File: app/validators/uuidValidator.js [hunk 3]
```diff
@@ -7,6 +7,4 @@ async function isUUIDValid(id) {
     }
 }
 
-const id = [param('id').custom(id => isUUIDValid(id))];
-
-module.exports = { id };
+module.exports = [param('id').custom(id => isUUIDValid(id))];
```
Comment (rafal-ksiazek-rmtcfm-com) [inline app/validators/uuidValidator.js]: Maybe it's better create more abstract validator:

module.export = (arg) => {
    return [
         param(arg).custom(id => isUUIDValid(id))
    ]
}

and then in some advanced router:

/users/:id/contracts/:contractId, [uuidValidator('id'), uuidValidator('contractId'), validate]

Comment (rafal-ksiazek-rmtcfm-com) [reply app/validators/uuidValidator.js]: Then change also text in promise.reject to 'must be valid UUID'

File: app/validators/uuidValidator.js (hunk 2 above)
Comment (daniel-pupczyk-rmtcfm-com) [reply app/validators/uuidValidator.js]: isn't returning boolean look better?
`custom(v => validate(v)).withMessage('Id must be valid UUID')`?

**Pull request merged**